app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingFragment.kt

```diff
@@ -149,34 +141,24 @@ class OnboardingFragment : Fragment() {
 
     @Composable
     fun KeyFeatureCard(iconId: Int, titleId: Int, descriptionId: Int) {
-        ConstraintLayout(modifier = Modifier.fillMaxWidth()) {
-            val (image, title, description) = createRefs()
-            Image(
-                painter = painterResource(iconId),
-                colorFilter = ColorFilter.tint(color = focusColors.onboardingKeyFeatureImageTint),
-                contentDescription = getString(R.string.app_name),
-                modifier = Modifier.constrainAs(image) { top.linkTo(parent.top) }
-            )
-            Text(
-                text = stringResource(id = titleId),
-                modifier = Modifier
-                    .constrainAs(title) {
-                        top.linkTo(image.top)
-                        start.linkTo(image.end, margin = 16.dp)
-                        bottom.linkTo(image.bottom)
-                    },
-                style = focusTypography.onboardingSubtitle
-            )
-            Text(
-                text = stringResource(id = descriptionId, getString(R.string.onboarding_short_app_name)),
-                modifier = Modifier.constrainAs(description) {
-                    top.linkTo(title.bottom)
-                    start.linkTo(title.start)
-                    end.linkTo(parent.end)
-                    width = Dimension.fillToConstraints
-                },
-                style = focusTypography.onboardingDescription
-            )
+        Row(modifier = Modifier.fillMaxWidth().padding(start=16.dp)){
+            Column() {
```

Comment:
On a first look I have mixed feelings about having a column for only one child. I will take a second look to see what we can do about that

Reply:
Sure, we will look into that

Reply:
@ionutbedregeanu , I think it's not necessary as you mentioned. We can remove the column.

```
@Composable
    fun KeyFeatureCard(iconId: Int, titleId: Int, descriptionId: Int) {
        Row(modifier = Modifier.fillMaxWidth().padding(start=16.dp)){
            Image(
                painter = painterResource(iconId),
                colorFilter = ColorFilter.tint(color = focusColors.onboardingKeyFeatureImageTint),
                contentDescription = getString(R.string.app_name),
            )

            Column(modifier = Modifier.padding(start = 16.dp)) {
                Text(
                    text = stringResource(id = titleId),
                    style = focusTypography.onboardingSubtitle
                )
                Text(
                    text = stringResource(id = descriptionId, getString(R.string.onboarding_short_app_name)),
                    style = focusTypography.onboardingDescription
                )
            }
        }
    }
```

Reply:
@ionutbedregeanu , I also added top 4.dp padding for the first text right of the icon

```
@Composable
    fun KeyFeatureCard(iconId: Int, titleId: Int, descriptionId: Int) {
        Row(modifier = Modifier.fillMaxWidth().padding(start=16.dp)){
            Image(
                painter = painterResource(iconId),
                colorFilter = ColorFilter.tint(color = focusColors.onboardingKeyFeatureImageTint),
                contentDescription = getString(R.string.app_name),
            )

            Column(modifier = Modifier.padding(start = 16.dp, top = 4.dp)) {
                Text(
                    text = stringResource(id = titleId),
                    style = focusTypography.onboardingSubtitle
                )
                Text(
                    text = stringResource(id = descriptionId, getString(R.string.onboarding_short_app_name)),
                    style = focusTypography.onboardingDescription
                )
            }
        }
    }
```